crates/sui-core/src/consensus_handler.rs

```diff
@@ -69,10 +70,13 @@ impl ExecutionState for ConsensusHandler {
         consensus_index: ExecutionIndices,
         serialized_transaction: Vec<u8>,
     ) {
-        let consensus_index =
-            Self::update_hash(&self.last_seen, consensus_index, &serialized_transaction);
-        let consensus_index = if let Some(consensus_index) = consensus_index {
-            consensus_index
+        let index = Self::update_hash(
```

Comment:
The reason I am changing this is because with the previous way we were overwriting the `consensus_index` value and assigning the value `None` when `Self::update_hash` returned such. That makes the debug log in line 82 print `Ignore consensus transaction at index None....`